### PR TITLE
Reference to string-specific directional information

### PR DESCRIPTION
**Pull request by sebastiankb**
solves #1416

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/pull/1499.html" title="Last updated on May 9, 2022, 2:05 PM UTC (6b842d4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-thing-description/1499/a76e087...6b842d4.html" title="Last updated on May 9, 2022, 2:05 PM UTC (6b842d4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1209,6 +1209,10 @@ <h2>Core Vocabulary Definitions</h2>
                 names are used in a <code>security</code>
                 name-value pair.</td><td>mandatory</td><td><a>Map</a> of <a href="#securityscheme"><code>SecurityScheme</code></a></td></tr>
 <tr class="rfc2119-table-assertion" id="td-vocab-profile--Thing"><td><code>profile</code></td><td>Indicates the WoT Profile mechanisms followed by this Thing Description and the corresponding Thing implementation.</td><td>optional</td><td><a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a> or <a>Array</a> of <a href="http://www.w3.org/TR/2012/REC-xmlschema11-2-20120405/#anyURI"><code>anyURI</code></a></td></tr>
+<tr class="rfc2119-table-assertion" id="td-vocab-schemaDefinitions--Thing"><td><code>schemaDefinitions</code></td><td>Set of named data schemas.
```
Comment (mlagally) [inline index.html:1212]: I have seen this change in several PRs. It seems unrelated to the issue this PR is solving.

File: index.html [hunk 2]
```diff
@@ -1288,47 +1291,15 @@ <h2>Core Vocabulary Definitions</h2>
...
-            <code>MultiLanguage</code> <a href="#dfn-map" class=
-            "internalDFN" data-link-type="dfn">Maps</a> <em class=
-            "rfc2119" title="MUST">MUST</em> include a script
-            subtag, so that an appropriate base direction can be
-            inferred.</span> An example is Azeri, which is written
-            LTR when Latin script is used (specified using
-            <code>az-Latn</code>) and RTL when Arabic script is
-            used (specified using <code>az-Arab</code>).</li>
-          </ul>
+          <p>To determine the base direction of all
```
Comment (mlagally) [inline index.html:1294]: That's a significant change and hard to review. Have the rules changed?

File: index.html (hunk 1 above)
Comment (JKRhb) [reply index.html:1212]: For some reason, the `schemaDefinitions` disappeared for some time from the TD document, but are now properly regenerated. Maybe a separate regeneration PR could be created and merged before this one, which would get rid of the unrelated changes?

**Comment by sebastiankb** [comment]
from today's TD call:
- decided not to merge
- we need to clarify if the reference should be an assertion. However, the referred document with the guideline is a W3C Note document. The question is, if we can embed this in a TD assertion. @aphillips @ashimura can you help here?
